UI Snippets/Login UI/index.html

```diff
@@ -29,4 +29,4 @@ <h1>User Login</h1>
         <p>By clicking on <strong>Login</strong>, you agree to our <u class=""TandC>Terms and conditions.</u></p>
```

Comment:
You have written the class after the quotes, please update that once

Reply:
Okay I will update that and will give a pull request. Any other modifications you want?